website/docs/r/log_search.html.markdown

```diff
@@ -10,18 +10,20 @@ Provides a Sumologic Log Search.
 
 ## Example Usage
 ```hcl
-data "sumologic_personal_folder" "personal_folder" {}
+data "sumologic_personal_folder" "personalFolder" {}
```

Comment:
Nitpick:

This is inconsistent with the casing used for for the other resources in this example, which all use snake case.